Solutions/RubrikSecurityCloud/Playbooks/RubrikUserIntelligenceAnalysis/azuredeploy.json

```diff
@@ -0,0 +1,1926 @@
+                                                "includeWhitelistedResults": true,
+                                                "isGroupPrincipalType": false,
+                                                "sort": {
+                                                    "sortBy": "RISK_LEVEL",
+                                                    "sortOrder": "DESC"
+                                                },
+                                                "timelineDate": "@{concat(formatDateTime(utcNow(),'yyyy-MM-dd'),'T00:00:00+00:00')}"
+                                            }
+                                        },
+                                        "headers": {
+                                            "Authorization": "Bearer @{variables('AccessToken')}"
+                                        },
+                                        "method": "POST",
+                                        "uri": "@{variables('BaseUrl')}/api/graphql"
+                                    }
+                                }
+                            },
+                            "runAfter": {
+                                "Initialize_AccessToken": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "else": {
+                                "actions": {
+                                    "Does_UserId_available": {
+                                        "actions": {
+                                            "Terminate": {
+                                                "runAfter": {},
+                                                "type": "Terminate",
+                                                "inputs": {
+                                                    "runStatus": "Succeeded"
+                                                }
+                                            }
+                                        },
+                                        "runAfter": {
+                                            "Until": [
+                                                "Succeeded"
+                                            ]
+                                        },
+                                        "else": {
+                                            "actions": {
+                                                "Add_comment_to_incident_(V3)": {
+                                                    "runAfter": {},
+                                                    "type": "ApiConnection",
+                                                    "inputs": {
+                                                        "body": {
+                                                            "incidentArmId": "@triggerBody()?['object']?['id']",
+                                                            "message": "<p>No user with email @{variables('Email')} is available in Rubrik.</p>"
```

Comment:
Here too

Reply:
@v-prasadboke I have implemented an alternative of using variables in incident comment but still facing arm-ttk failures.